crawler_mcp/.env.example

```
@@ -0,0 +1,95 @@
+# -----------------------------
+SERVER_HOST=127.0.0.1
+SERVER_PORT=8010
+DEBUG=false
+LOG_LEVEL=INFO
+
+# -----------------------------
+# Qdrant (Vector DB)
+# -----------------------------
+QDRANT_URL=http://localhost:6333
+QDRANT_API_KEY=
+QDRANT_COLLECTION=crawlerr_documents
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The QDRANT_COLLECTION key should go before the QDRANT_URL key

(UnorderedKey)